### PR TITLE
Add style utilities package to alias styles package

### LINKED ISSUE #24569: @fluentui/style-utilities (re-exports @uifabric/styling)

### PR DESCRIPTION
**Pull request by micahgodbolt**
Fixes #24569

### CODE REVIEW

File: packages/style-utilities/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (GeoffCoxMSFT) [inline packages/style-utilities/package.json:1]: Need the beachball config entry here as well right?

File: packages/style-utilities/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  "dependencies": {
+    "@microsoft/load-themed-styles": "^1.10.26",
+    "@fluentui/theme": "^1.7.7",
+    "@uifabric/styling": "^7.21.2",
+    "@uifabric/merge-styles": "^7.19.2",
+    "@uifabric/set-version": "^7.0.24",
+    "@uifabric/utilities": "^7.35.0",
+    "tslib": "^1.10.0"
+  }
```
Comment (GeoffCoxMSFT) [inline packages/style-utilities/package.json:43]:
```suggestion
    "beachball": {
      "tag": "lts-7"
    }
  }
```

**Comment by fabricteam** [comment]
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">7.0 Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/BaseButton-mount_base.html">699</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/BaseButton-mount.html">674</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Breadcrumb-mount_base.html">32749</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Breadcrumb-mount.html">31874</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Checkbox-mount_base.html">1232</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Checkbox-mount.html">1292</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/CheckboxBase-mount_base.html">1012</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/CheckboxBase-mount.html">1074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/ChoiceGroup-mount_base.html">4093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/ChoiceGroup-mount.html">4168</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/ComboBox-mount_base.html">706</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/ComboBox-mount.html">730</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/CommandBar-mount_base.html">6095</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/CommandBar-mount.html">6332</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/ContextualMenu-mount_base.html">10970</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/ContextualMenu-mount.html">11095</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DefaultButton-mount_base.html">927</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DefaultButton-mount.html">912</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DetailsRow-mount_base.html">2856</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DetailsRow-mount.html">2887</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DetailsRowFast-mount_base.html">2836</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DetailsRowFast-mount.html">2848</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DetailsRowNoStyles-mount_base.html">2671</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DetailsRowNoStyles-mount.html">2832</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Dialog-mount_base.html">1798</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Dialog-mount.html">1738</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DocumentCardTitle-mount_base.html">1348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/DocumentCardTitle-mount.html">1346</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Dropdown-mount_base.html">1876</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Dropdown-mount.html">1872</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/FocusTrapZone-mount_base.html">1301</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/FocusTrapZone-mount.html">1327</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/FocusZone-mount_base.html">1522</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/FocusZone-mount.html">1344</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/IconButton-mount_base.html">1309</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/IconButton-mount.html">1318</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Label-mount_base.html">284</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Label-mount.html">266</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Layer-mount_base.html">1432</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Layer-mount.html">1640</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Link-mount_base.html">330</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Link-mount.html">340</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/MenuButton-mount_base.html">1140</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/MenuButton-mount.html">1170</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/MessageBar-mount_base.html">1668</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/MessageBar-mount.html">1675</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Nav-mount_base.html">2705</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Nav-mount.html">2609</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/OverflowSet-mount_base.html">1144</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/OverflowSet-mount.html">1128</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Panel-mount_base.html">1130</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Panel-mount.html">1160</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Persona-mount_base.html">725</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Persona-mount.html">761</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Pivot-mount_base.html">1168</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Pivot-mount.html">1132</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/PrimaryButton-mount_base.html">1044</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/PrimaryButton-mount.html">1068</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Rating-mount_base.html">6187</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Rating-mount.html">6176</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/SearchBox-mount_base.html">1003</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/SearchBox-mount.html">1087</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Shimmer-mount_base.html">1953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Shimmer-mount.html">1993</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Slider-mount_base.html">1193</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Slider-mount.html">1161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/SpinButton-mount_base.html">3971</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/SpinButton-mount.html">4015</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Spinner-mount_base.html">323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Spinner-mount.html">325</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/SplitButton-mount_base.html">2440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/SplitButton-mount.html">2602</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Stack-mount_base.html">418</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Stack-mount.html">420</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/StackWithIntrinsicChildren-mount_base.html">1316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/StackWithIntrinsicChildren-mount.html">1317</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/StackWithTextChildren-mount_base.html">3996</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/StackWithTextChildren-mount.html">3737</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/SwatchColorPicker-mount_base.html">8426</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/SwatchColorPicker-mount.html">8393</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/TagPicker-mount_base.html">2356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/TagPicker-mount.html">2228</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/TeachingBubble-mount_base.html">39041</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/TeachingBubble-mount.html">38978</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Text-mount_base.html">315</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Text-mount.html">303</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/TextField-mount_base.html">1032</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/TextField-mount.html">1173</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Toggle-mount_base.html">709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/Toggle-mount.html">618</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/button-mount_base.html">80</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24572/perf-test/button-mount.html">98</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by micahgodbolt** [comment]
need to update beachball groups before merging this

**Review by GeoffCoxMSFT — APPROVED** [review]

File: scripts/beachball/index.ts [hunk 3]
```diff
@@ -9,6 +9,11 @@ export const config: BeachballConfig = {
       include: ['packages/office-ui-fabric-react', 'packages/react'],
       disallowedChangeTypes: ['major'],
     },
+    {
+      name: 'Charting',
+      include: ['packages/charting', 'stub-packages/react-charting'],
```
Comment (TristanWatanabe) [inline scripts/beachball/index.ts:14]: Was this intentionally meant to be `react-charting` or should this be `style-utilities`?

**Review by TristanWatanabe — COMMENTED** [review]
One nit and a question

File: scripts/beachball/index.ts (hunk 3 above)
Comment (micahgodbolt) [inline scripts/beachball/index.ts:14]:
```suggestion
      name: 'Styling',
      include: ['packages/styling', 'stub-packages/style-utilities'],
```

**Pull request merged**